pom.xml

```diff
@@ -193,6 +193,19 @@
       <tag>HEAD</tag>
   </scm>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jenkins-ci.tools</groupId>
+        <artifactId>maven-hpi-plugin</artifactId>
+        <configuration>
+          <!-- JENKINS-47074: Build ID precedes display names. -->
+          <compatibleSinceVersion>1.40</compatibleSinceVersion>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
```

Comment:
This change may cause regressions if `yyyy-MM-dd_HH-mm-ss` is used for display names.
(I don't think it's often the case, though).